File: java/program/Memory.java
class Memory {
    private int kapasitasGB;
    private int kecepatanMHz;

    public Memory() {
    }

    public Memory(int kapasitasGB, int kecepatanMHz) {
        this.kapasitasGB = kapasitasGB;
        this.kecepatanMHz = kecepatanMHz;
    }

    public void setKapasitasGB(int kapasitasGB) {
        this.kapasitasGB = kapasitasGB;
    }

    public void setKecepatanMHz(int kecepatanMHz) {
        this.kecepatanMHz = kecepatanMHz;
    }

    public int getKapasitasGB() {
        return this.kapasitasGB;
    }

    public int getKecepatanMHz() {
        return this.kecepatanMHz;
    }
}

File: python/program/Memory.py
class Memory:
    __kapasitasGB = 0
    __kecepatanMHz = 0

    def __init__(self, kapasitasGB, kecepatanMHz):
        self.__kapasitasGB = kapasitasGB
        self.__kecepatanMHz = kecepatanMHz

    def set_kapasitasGB(self, kapasitasGB):
        self.__kapasitasGB = kapasitasGB

    def set_kecepatanMHz(self, kecepatanMHz):
        self.__kecepatanMHz = kecepatanMHz

    def get_kapasitasGB(self):
        return self.__kapasitasGB

    def get_kecepatanMHz(self):
        return self.__kecepatanMHz
